pages/glossary/content-that-becomes-visible.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Content that becomes visible
+key: content-that-becomes-visible
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+The _content that becomes visible_ is the [root][] element, if one exists, of the [tree][] that contains all elements that, as result of an [event][] [firing][], meet any of the following:
+
+- the element has [visible text content][] and is added to the [web page][] where the [event][] was [fired][firing]; or
+- the element [attributes][]' [values][] are modified in a way that cause some or all of its [text nodes][] to become [visible][].
```

Comment:
Visible text content isn't bound to the closest ancestor. Any text nodes added to the page results in the HTML element having visible text content added.

How about something like:

```suggestion
has visible text content that did not exist on the page prior to the event firing.
```

Reply:
We don't need to worry about the bounding because this tree only contains new elements (therefore elements that were there already are not considered) and elements that have had their attributes modified (therefore if their text nodes changed without a change in their attributes, they are not considered)